### PR TITLE
Improved the install-sdk target

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
This will now skip the download of the SDK if the same version already exists at the destination.

File: .ci/install-sdk.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+DEST="${GOPATH}/bin/operator-sdk"
+
+function install_sdk() {
+    echo "Downloading the operator-sdk ${SDK_VERSION} into ${DEST}"
+    curl https://github.com/operator-framework/operator-sdk/releases/download/${SDK_VERSION}/operator-sdk-${SDK_VERSION}-x86_64-linux-gnu -sLo ${DEST}
+    chmod +x ${DEST}
+}
+
+if [ ! -d ${GOPATH}/bin/ ]; then
```
Comment (pavolloffay) [inline .ci/install-sdk.sh]: `mkdir -p` is noop and does not fail if the dir exists. The check does not seem to be necessary.

File: .ci/install-sdk.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+DEST="${GOPATH}/bin/operator-sdk"
+
```
Comment (pavolloffay) [inline .ci/install-sdk.sh]: Shouldn't it also define the default for `SDK_VERSION`? This way the script would work without the caller knowing the configuration options.

Comment (jpkrohling) [reply .ci/install-sdk.sh]: If we ever have someone to use this script manually, we can change it. Right now, it's easiest to just use the same target as before: `make install-sdk`

Comment (pavolloffay) [reply .ci/install-sdk.sh]: fair enough. Defining all the prerequisites helps with portability and readability. But it's just a ci script.

**Pull request merged**